mmengine/runner/runner.py

```diff
@@ -1670,6 +1692,14 @@ def test(self) -> dict:
         Returns:
             dict: A dict of metrics on testing set.
         """
+        if is_model_wrapper(self.model):
+            ori_model = self.model.module
+        else:
+            ori_model = self.model
+        assert hasattr(ori_model, 'test_step'), (
+            'If you want to train your model, please make sure your model '
+            'has implemented `test_step`.')
```

Comment:
This checking is not required because `runner.test` will call `self.test_loop.run` soon.